src/index.keyboardNav.test.js

```diff
@@ -161,3 +161,47 @@ test('should set current focus as selected on tab out for simpleSelect', t => {
   triggerOnKeyboardKeyDown(wrapper, ['ArrowDown', 'ArrowRight', 'ArrowRight', 'Tab'])
   t.deepEqual(wrapper.state().tags[0].label, 'ccc 1')
 })
+
+test('should scroll on keyboard navigation', t => {
```

Comment:
Love these tests! 👍

Reply:
I need to add one ensuring scroll is intact on regular use also.
Everthing gets a bit harder without a real dom though 😆

Reply:
Commenting the following line https://github.com/dowjones/react-dropdown-tree-select/blob/f7b7dd5339f4879207587c5a0d9ea3884154786d/src/tree/index.js#L53 fails the newly added test.